BREAKING.md

```diff
@@ -33,6 +34,33 @@ with this one:
 ### IFluidLodable.url is removed
 `url` property is removed. If you need a path to an object (in a container), you can use IFluidLoadable.handle.absolutePath instead.
 
+### Loader Constructor Changes
+The loader constructor has changed to now take a props object, rather than a series of paramaters. This should make it easier to construct loaders as the optional services can be easily excluded.
+
+Before:
+``` typescript
+    const loader = new Loader(
+        urlResolver,
+        documentServiceFactory,
+        codeLoader,
+        { blockUpdateMarkers: true },
+        {},
+        new Map(),
+    );
+```
+
+After:
+``` typescript
+    const loader = new Loader({
+        urlResolver,
+        documentServiceFactory,
+        codeLoader,
+    });
+```
+
+if for some reason this change causes you problems, we've added a deprecated `Loader._create` method that has the same parameters as the previous constructor which can be used in the interim.
```

Comment:
Seems to be not necessary. I.e. Given that code needs to be changed either way, there does not seem to be good reason to move to something that we want to deprecate right away.

Reply:
yeah. one of our partners is doing some odd things, so this should make it easier on them